lib/views/video_view/custom_video_control_bar.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:flutter/material.dart';
+import 'package:gocast_mobile/base/networking/api/gocast/api_v2.pb.dart';
+
+class CustomVideoControlBar extends StatelessWidget {
+  final Function(String, Stream) onMenuSelection;
+  final VoidCallback onToggleChat;
+  final VoidCallback onOpenQuizzes;
+  final Stream currentStream;
+
+  const CustomVideoControlBar({
+    super.key,
+    required this.onMenuSelection,
+    required this.onToggleChat,
+    required this.onOpenQuizzes,
+    required this.currentStream,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    bool isIOS = Theme.of(context).platform == TargetPlatform.iOS;
+    return Container(
+      padding: const EdgeInsets.symmetric(vertical: 10, horizontal: 16),
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+        children: [
+          // Icons on the left
+          Row(
+            children: [
+              IconButton(
+                icon: const Icon(Icons.messenger_outline_outlined),
+                onPressed: onToggleChat,
+              ),
+              IconButton(
+                icon: const Icon(Icons.quiz_outlined),
+                onPressed: onOpenQuizzes,
+              ),
+            ],
+          ),
+
+          // Three-dot menu on the right
```

Comment:
I'd suggest only showing the icons of the avaiable views (e.g., if a stream only has presentation view (and therefore only that URL is set and all other URL-fields are either empty or null, only the presentation view icon is shown)

Reply:
Done ✅ 
It was also causing the previous issue.